### DISCUSSION TITLE
Fleather and Flutter not compatible?

### DISCUSSION THREAD

**Discussion by fgonzalez-tenco** [fleather-editor/fleather#254] [Q&A] [answered]
Hello! 
I'm working with a flutter app and I'm having some issues while trying to build my application:
![image](https://github.com/fleather-editor/fleather/assets/67757445/e4f7f113-a9d7-4e2d-adb7-ca133b7c591e)

These are the versions i'm using:

- Fleather -> 1.11.0
- Flutter ->  3.16.9 (Channel stable)

Also, if I try to use the newest version of Fleatter (1.13.2 or 1.14.0+1) I got an error on the `flutter pub get` command: 
![image](https://github.com/fleather-editor/fleather/assets/67757445/45198d56-eed4-47af-ac9c-931d4ae1c892)

😞

**Comment by Amir-P** [comment]
Sorry for the inconvenience. We are using latest `intl` package but `flutter_localization` is not, and this is causing conflict. Please add `intl: 0.18.1` to your `dependency_overrides`.

```yaml
dependency_overrides:
  intl: ^0.18.1
```

**Comment by Amir-P** [comment, marked as answer]
Please upgrade to the newest version. It should work now.

```yaml
fleather: ^1.14.1
```
